src/js/apps/patients/worklist/worklist_app.js

```diff
@@ -253,6 +279,22 @@ export default App.extend({
 
     return _.union(sortDueOptions, sortUpdateOptions);
   },
+  showDeleteSuccess(itemCount) {
+    if (this.getState().isFlowType()) {
+      Radio.request('alert', 'show:success', renderTemplate(hbs`{{formatMessage  (intlGet "patients.worklist.worklistApp.bulkDeleteFlowsSuccess") itemCount=itemCount}}`, { itemCount }));
```

Comment:
I don't like the `renderTemplate(hbs`...intlGet` pattern.. it's so cumbersome, but I can't seem find an alternative.

Reply:
Yeah, I wasn't a huge fan of it either. The other option would be to not display the number of items, but that seems like a degradation in the user experience. To clean things up, I could move the templates to the views file and import them here.

Reply:
Yeah I think defining the templates over there is better anyhow.. keep as much of intl out of the apps as possible.